alertmanager_gchat_integration/app.py

```diff
@@ -70,10 +70,11 @@ def post_alerts():
         text_alert = J2_TEMPLATE_ENGINE.render(render_payload)
 
         # Post request.
+        message_type = 'cards' if CONFIG['app']['notification'].get('use_cards') else 'text'
         post_request_details = requests.post(
             notification_url,
             json={
-                'text': text_alert
+                message_type: text_alert
```

Comment:
if `CONFIG['app']['notification'].get('use_cards')` is true, `text_alert` needs to be converted to a `dict` using `json.loads(text_alert)`.